safe_control_gym/envs/gym_pybullet_drones/quadrotor.py

```diff
@@ -1,11 +1,12 @@
-"""1D and 2D quadrotor environment using PyBullet physics.
+'''1D and 2D quadrotor environment using PyBullet physics.
```

Comment:
It's 3D too now, we should update this and below

Reply:
Done